include/deal.II/simplex/quadrature_lib.h

```diff
@@ -0,0 +1,48 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_tet_quadrature_lib_h
+#define dealii_tet_quadrature_lib_h
+
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/quadrature_lib.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Simplex
```

Comment:
namespace documentation? Otherwise it won't show up in doxygen.

Reply:
One of the other PRs contains the namespace docu!